java/ql/src/Security/CWE/CWE-297/InsecureJavaMail.ql

```diff
@@ -0,0 +1,23 @@
+/**
+ * @name Insecure JavaMail SSL Configuration
+ * @description Java application configured to use authenticated mail session
+ *              over SSL does not validate the SSL certificate to properly
```

Comment:
The preferred syntax being "Syntax X causes behavior Y.", I'd suggest the following here.
Feel free to tweak the wording if it is not technically correct.
```suggestion
 *              over SSL without certificate validation
```

Reply:
It's great, thanks!